scripts/start

```diff
@@ -124,5 +127,5 @@ show_conflict_warning() {
   fi
 
   # Start the service, replacing existing containers unless quick flag received
-  compose up --force-recreate $@
+  compose up --force-recreate $@ ${service}
```

Comment:
the `test` and `start` scripts both need message queues available, so I've pushed the service into the base file.
This is not required when linking so I've added the service name to the compose up to only start the core service when the -l switch is passed.